.github/codeql/codeql-config.yml

```
@@ -0,0 +1,4 @@
+paths:
+  - src
+  - profiles
+  - js
```

Maybe this instead? This will include tools and worker too.
```suggestion
paths-ignore:
  - builds
  - tests
```

---

Good point.